stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/components/MessageInput.kt

```diff
@@ -89,24 +90,21 @@ public fun MessageInputOptions(
  * Input field for the Messages/Conversation screen. Allows label customization, as well as handlers
  * when the input changes.
  *
- * @param value Current value of the input.
- * @param attachments Currently selected and visible attachments.
- * @param activeAction Currently active action (for [Edit] UI).
+ * @param messageInputState The state of the input.
  * @param onValueChange Handler when the value changes.
  * @param onAttachmentRemoved Handler when the user removes a selected attachment.
  * @param modifier Modifier for styling.
  * @param label Composable function that represents the label UI, when there's no input/focus.
  */
 @Composable
 public fun MessageInput(
-    value: String,
-    attachments: List<Attachment>,
-    activeAction: MessageAction?,
```

Comment:
I feel like we're losing something here by making the stateless component take a single state parameter instead of the individual ones. It was neater the other way imo (having `value` / `onValueChange` especially).